### ISSUE TITLE
Request - Ability to Save MySims/Kingdom Packages 📦

### ISSUE THREAD

**Issue by ILoveMySimsOk** [Olivercomet/MorcuTool#6]
Hello! Firstly, I _really_ wanted to **thank you** for caring about MySims and keeping this community alive, after so many years. You making this tool helps modders work a lot.
I wanted to ask, if it's not too much trouble, or if it's not too difficult to implement, for you to add the feature of saving the `.package` of MySims and MySims Kingdom, doing this would be a **great relief for modders**, as in addition to adding the possibility of modding MySims Kingdom would make modding MySims itself easier! 💙

**Comment by bartek901** [comment]
Yeah, it would be great to save packages for PC mysims... there is no way to pack it when you change lua files, as s3pe works only with textures, but as the time passed i guess this project is dead, unfortunately :(

**Comment by Olivercomet** [comment]
Sorry, but I don't really have the time to work on this anymore :(

Morcutool was built piecemeal when I was a less experienced programmer, and
would probably need a massive overhaul for package saving to be added in a
sensible way, unfortunately.
